frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -431,6 +441,15 @@ export default function InteractiveWaveformModal({
       endTime: existingData.editableStartEndTimes.endTime,
     });
     setPeakValleyWindows(existingData.peakValleyWindows);
+    setPeakSlope(existingData.peakSlope);
+    setPeakYIntercept(existingData.peakYIntercept);
+    setPeakY1(existingData.peakY1);
+    setPeakY2(existingData.peakY2);
+    setValleySlope(existingData.valleySlope);
+    setValleyYIntercept(existingData.valleyYIntercept);
+    setValleyY1(existingData.valleyY1);
+    setValleyY2(existingData.valleyY2);
+    getWaveformData(true, selectedWell);
```

Comment:
what happens in the changelog for this? and the "undo" feature?

Reply:
and the save feature?

Reply:
The save feature saves all the slopes, intercepts, y1 and y2 for each line in each well. On load a modal shows up to confirm loading from last session or start over.

The undo feature only undoes changes to wells. I need to add the limit lines to the change log and undo feature.